.github/workflows/Windows.yml

```diff
@@ -240,4 +243,6 @@ jobs:
          s3_id: ${{ secrets.S3_ID }}
          s3_key: ${{ secrets.S3_KEY }}
          signing_pk: ${{ secrets.DUCKDB_EXTENSION_SIGNING_PK }}
-         unittest_script: python3 scripts/run_tests_one_by_one.py ./build/release/test/Release/unittest.exe
\ No newline at end of file
+         run_tests: ${{ !startsWith(github.ref, 'refs/tags/v') }}
+         run_autoload_tests: ${{ !startsWith(github.ref, 'refs/tags/v') }}
```

Comment:
```suggestion
```

This I would remove, given default it's already 0 it will only do more testing instead of less.
Then it might be a good idea, but probably there are some reasons as to why this is enabled only for OSX.

Reply:
Default for `run_autoload_tests` is 1, no?

Reply:
You are right, sorry for the noise